public/app/plugins/datasource/cloudwatch/tracking.ts

```diff
@@ -0,0 +1,93 @@
+import { DashboardLoadedEvent } from '@grafana/data';
+import { reportInteraction } from '@grafana/runtime';
+
+import { isCloudWatchLogsQuery, isCloudWatchMetricsQuery, isCloudWatchQuery } from './guards';
+import pluginJson from './plugin.json';
+import { CloudWatchMetricsQuery, MetricEditorMode, MetricQueryType } from './types';
+
+interface CloudWatchOnDashboardLoadedTrackingEvent {
+  grafana_version?: string;
+  dashboard_id?: string;
+  org_id?: number;
+  user_id?: number;
+  grafanaVersion?: string;
```

Comment:
you should use snake case for events props
```suggestion
  grafana_version?: string;
```

Reply:
Oh thanks! It's also duplicated. :|